docs/examples.rst

```diff
@@ -1,9 +1,27 @@
 
 Examples
---------
+=========
 
-Explain any pre-reqs needed by users in order to run the examples below
+Pre Requisites
+---------------
 
+Before we run the below examples, there are a few things we need to have up and running. 
+
+GlobusConnectPersonal
+######################
+
+To run tools like ``gladier_tools.globus.Transfer``, we need access to Globus endpoints. One can use tutorial endpoints 1 and 2 `here <https://app.globus.org/file-manager>`_.
+However, setting up your own endpoint gives you more control over the data you transfer, which brings us to GlobusConnectPersonal. Follow the instructions `here <https://www.globus.org/globus-connect-personal>`_ to set up your own GCP endpoint, then navigate to `Globus Webapp endpoints <https://app.globus.org/endpoints>`_ to see details about the endpoint. 
+
+
+funcx-endpoint
+###############
+
+Another thing we need to run flows are an endpoint to run the funcx-functions on. There is a tutorial endpoint that can be used but it is generally preferred to set up a funcx-endpoint. 
+Follow the instructions `here <https://funcx.readthedocs.io/en/latest/endpoints.html>`_ to set up your own endpoint.
```

Comment:
Linking to the docs is a good idea for the details on how to setup a funcx endpoint

It would be good to list an example for what the user should have after running through the FuncX doc. I think a good example would be listing their endpoint, like so:

```
$ funcx-endpoint list
+---------------+--------------+--------------------------------------+
| Endpoint Name |    Status    |             Endpoint ID              |
+===============+==============+======================================+
| local         | Ready        | 54db2d4b-5e89-4398-8688-d0e9d5d37fcb |
+---------------+--------------+--------------------------------------+
```